src/main/java/com/iexec/core/security/JwtTokenProvider.java

```diff
@@ -58,46 +74,50 @@ public String resolveToken(String token) {
         return null;
     }
 
-    /*
-     * IMPORTANT /!\
-     * Having the same validity duration for both challenge
-     * and jwtoken can cause a problem. The latter should be
-     * slightly longer (in minutes). In this case the challenge
-     * is valid for 60 minutes while jwtoken stays valid
-     * for 65 minutes.
-     * 
-     * Problem description:
-     *  1) jwtString expires
-     *  2) worker gets old challenge
-     *  3) old challenge expires
-     *  4) worker tries logging with old challenge
+    /**
+     * Checks if a JWT is valid.
+     * <p>
+     * A valid JWT must:
+     * <ul>
+     * <li>be signed with the scheduler private key
+     * <li>not be expired
+     * <li>contain valid address and challenge values respectively in audience and subject claims
```

Comment:
Can you remove/update these comments (mostly challenge-related)

Reply:
Sure, let me know if it is better in 52483222